products/pan-os.md

```diff
@@ -37,6 +40,6 @@ releases:
     eol: 2020-06-30
     release: 2016-03-29
 ---
-> [Palo Alto Networks](https://www.paloaltonetworks.com/) [PAN-OS](https://docs.paloaltonetworks.com/pan-os) is the software that runs all Palo Alto Networks next-generation firewalls. These firewalls are deployed as physical appliances, virtual machines, containers, and cloud services.
+[Palo Alto Networks](https://www.paloaltonetworks.com/) [PAN-OS](https://docs.paloaltonetworks.com/pan-os) is the software that runs all Palo Alto Networks next-generation firewalls. These firewalls are deployed as physical appliances, virtual machines, containers, and cloud services.
```

Comment:
We use a blockquote as a stylistic choice for product descriptions, so it's [consistent across all products](https://github.com/endoflife-date/endoflife.date/blob/master/CONTRIBUTING.md?plain=1#L148-L158). Let's undo this and merge.

```suggestion
> [Palo Alto Networks](https://www.paloaltonetworks.com/) [PAN-OS](https://docs.paloaltonetworks.com/pan-os) is the software that runs all Palo Alto Networks next-generation firewalls. These firewalls are deployed as physical appliances, virtual machines, containers, and cloud services.
```

Reply:
Fair doos! :)